modules/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,79 @@
+= Configuring {rh-openstack} Availability Zones type
+
+You have installed a cluster on {rh-openstack-first} or on existing {product-title} cluster version 4.13 or lower, and the root volumes are deployed without updating Availability Zones (AZ) in the `install-config.yaml` file. 
+
+This makes the root volume availability zone to be taken from the compute availability zone, which is not available in cinder. The control plane machine cannot be created. 
+
+[NOTE]
+====
+For the {product-title} clusters version 4.14 and higher, the installers validate the root AZ in `install-congif.yaml` file and ensures the deployment to be continued.
+====
+
+For the clusters deployed before the {product-title} version 4.14, you need to manually update the control plane machine resources, so that they are updated on the instances.
+
+[NOTE]
+====
+Manually editing control plane machine resources does not update the control plane instances. However, it is necessary in order for the `cluster-control-plane-machine-set-operator` to correctly generate a `ControlPlaneMachineSet` for your cluster.
+====
+
+. Update the `ProviderSpec`, for all the control plane machines matching the environment, and set the property `rootVolume.availabilityZone` of `spec.providerSpec` to the value of the actual volume AZ (example with `master-0`):
++
+[source,terminal]
+----
+oc edit machine/<cluster_id>-master-0 -n openshift-machine-api
+<make edits>
+----
++
+[source,Sample Output]
+----
+providerSpec:
+  value:
+    apiVersion: machine.openshift.io/v1alpha1
+    availabilityZone: az0
+      cloudName: openstack
+    cloudsSecret:
+      name: openstack-cloud-credentials
+      namespace: openshift-machine-api
+    flavor: m1.xlarge
+    image: rhcos-4.14
+    kind: OpenstackProviderSpec
+    metadata:
+      creationTimestamp: null
+    networks:    
+    - filter: {}
+      subnets:  
+      - filter:
+          name: refarch-lv7q9-nodes
+          tags: openshiftClusterID=refarch-lv7q9
+    rootVolume:
+        availabilityZone: nova <--- Add the zone name here
+        diskSize: 30
+        sourceUUID: rhcos-4.12
+        volumeType: fast-0
+    securityGroups:
+    - filter: {}
+      name: refarch-lv7q9-master
+    serverGroupName: refarch-lv7q9-master
+    serverMetadata:
+      Name: refarch-lv7q9-master
+      openshiftClusterID: refarch-lv7q9
+    tags:
+    - openshiftClusterID=refarch-lv7q9
+    trunk: true
+    userDataSecret:
+      name: master-user-data
+----
+
+. In case you updated your control plane machine resources after install, follow these steps. 
+.. In your {rh-openstack} cluster, find the AZ of the root volumes for your control plane machines and set it in the `availabilityZone` property of all the control plane machines.
+
+[NOTE]
+====
+Once all the control plane machines have a root volume AZ, your control plane is ready to be managed by the Cluster Control Plane Machine Set operator and a `ControlPlaneMachineSet` (CPMS) will be created.
+====
```

Comment:
```suggestion
After all of the control plane machines have a root volume availability zone, your control plane is ready to be managed by the Cluster Control Plane Machine Set Operator. The Operator will generate a control plane machine set for you.
```

Verify for accuracy. I'm just in editing mode.

Reply:
Ack. Technically accurate.